lib/tasks/tasks.rake

```diff
@@ -184,7 +185,14 @@ namespace :inferno do |_argv|
       csv << ['', '', '', '', '']
       csv << ['Test ID', 'Reference', 'Sequence/Group', 'Test Name', 'Required?', 'Reference URI']
       flat_tests.each do |test|
-        csv << [test[:test_id], test[:ref], test[:sequence].split('::').last, test[:name], test[:sequence_required] && test[:required], test[:url]]
+        csv << [
+          test[:test_id],
+          test[:ref],
```

Comment:
Is `ref` actually a thing?

Reply:
I stick a `pry` in here and it wasn't populated for any tests.

Reply:
This is a placeholder for a reference back to regulation, if applicable.  Presumably this should be filled out with NPRM content for the Program Edition at this point, but we haven't done that.  I created a ticket for doing this when the rule is finalized.